Updated event and endpoint information (#503)
casper-network/docs: source/docs/casper/dapp-dev-guide/building-dapps/monitoring-events.md

# Monitoring and Consuming Events

> The Casper node streams deploy execution effects and finality signatures through an SSE server. The default configuration of the Casper node provides event streaming on the `/events` endpoint and the port specified as the `event_stream_server.address` in the node's *config.toml*, which is by default `9999` on [Testnet](https://testnet.cspr.live/tools/peers) and [Mainnet](https://cspr.live/tools/peers). The URLs for different types of events are:

Reviewer: ```suggestion
The Casper node streams deploy execution effects and finality signatures through an SSE server. The default configuration of the Casper node provides event streaming via the port specified as the `event_stream_server.address` in the node's *config.toml*, which is by default `9999` on [Testnet](https://testnet.cspr.live/tools/peers) and [Mainnet](https://cspr.live/tools/peers). The URLs for different types of events are:
```

- `/events/deploys`

>     The DeployAccepted events are emitted on `/events/deploys` endpoint. This means when a deploy is received by the node and it passes the first set of validity checks, it is stored locally, gossiped to peers and enqueued for inclusion in a block - at this point the DeployAccepted event is emitted.

Reviewer: ```suggestion

    `DeployAccepted` events are emitted on the `/events/deploys` endpoint. When a deploy is received by the node, if it passes the first set of validity checks, once it is stored locally the `DeployAccepted` event is emitted.

```

- `/events/sigs`

>     The FinalitySignature is emitted on `/events/sigs` endpoint, whenever a new finality signature is received.

Reviewer: ```suggestion

    `FinalitySignature` events are emitted on the `/events/sigs` endpoint, whenever a new finality signature is received.

```

- `/events/main` 

>     -   The other events such as BlockAdded, DeployProcessed, DeployExpired, Fault and Step are emitted on the `/events/main` endpoint.

Reviewer: ```suggestion

    All other events are emitted on the `/events/main` endpoint as follows:

```

>         -   BlockAdded - This event is emitted whenever a new block is added to the blockchain.
>         -   DeployProcessed - This event is emitted when the given deploy has been executed, committed and forms part of the given block.
>         -   DeployExpired - This event is emitted if a deploy is not added to a block for processing by a validator before the deploy's time to live (TTL) expires.
>         -   Fault - This event is emitted if there is a validator error.
>         -   Step - This event is emitted when the execution effects produced by running the auction contract at the end of every era.
>         -   Shutdown - This event is emitted when the node is about to shut down, usually for an upgrade.
>         -   ApiVersion - This is always the first event emitted when a new client connects to the SSE server, specifying the API version of the server.

Reviewer: ```suggestion
    - `ApiVersion` is always the first event emitted when a new client connects to the SSE server, specifying the API version of the server
    - `BlockAdded` events are emitted whenever a new block is added to the blockchain
    - `DeployProcessed` events are emitted when the given deploy has been executed, committed and forms part of the given block
    - `DeployExpired` events are emitted when deploys become no longer valid to be executed or added to a block due to their times to live (TTLs) expiring
    - `Fault` events are emitted if there is a validator error
    - `Step` events are emitted at the end of every era and contain the execution effects produced by running the auction contract's `step` function
    - `Shutdown` events are emitted when the node is about to shut down, usually for an upgrade
```
